OpenShift-Installation/Openshift-Installation.rst

```diff
@@ -129,17 +103,11 @@ Setup
    
    For Nuage releases 5.2.1, go `here <https://docs.openshift.com/container-platform/3.6/install_config/install/host_preparation.html>`_. 
 
-   For Nuage releases 5.2.2 & later, go `here <https://docs.openshift.com/container-platform/3.7/install_config/install/host_preparation.html>`_. 
+   For Nuage releases 5.2.2 to 5.3.3, go `here <https://docs.openshift.com/container-platform/3.7/install_config/install/host_preparation.html>`_.
```

Comment:
I think it would be better to remove these lines for 5.2.1, 5.2.2 to 5.3.3, and just, at the top of the document, add a section:
```
Installation with older Nuage VSP releases
===============================

* For installation instructions for Nuage VSP 5.3.3, follow this guide: https://github.com/nuagenetworks/nuage-kubernetes/blob/v5.3.3-1/OpenShift-Installation/Openshift-Installation.rst
* For installation instructions for Nuage VSP 5.3.2, follow this guide: https://github.com/nuagenetworks/nuage-kubernetes/blob/v5.3.2-1/OpenShift-Installation/Openshift-Installation.rst
* For installation instructions for Nuage VSP 5.3.1, follow this guide: https://github.com/nuagenetworks/nuage-kubernetes/blob/v5.3.1-1/OpenShift-Installation/Openshift-Installation.rst
* For installation instructions for Nuage VSP 5.2.3, follow this guide: https://github.com/nuagenetworks/nuage-kubernetes/blob/v5.2.3-1/OpenShift-Installation/Openshift-Installation.rst
* For installation instructions for Nuage VSP 5.2.2, follow this guide: https://github.com/nuagenetworks/nuage-kubernetes/blob/v5.2.2-1/OpenShift-Installation/Openshift-Installation.rst
* For installation instructions for Nuage VSP 5.2.1, follow this guide: https://github.com/nuagenetworks/nuage-kubernetes/blob/v5.2.1-1/OpenShift-Installation/Openshift-Installation.rst
```
That way, you can point them to the right Git Tag